pages/1.11/release-notes/1.11.8/index.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.11.8
+title: Release Notes for 1.11.8
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.11.8
+---
+
+DC/OS 1.11.8 was released on December 6, 2018. 
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.11.7/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+[button color="light" href="https://support.mesosphere.com/hc/en-us/articles/213198586"]Download DC/OS Enterprise[/button]
+
+DC/OS 1.11.8 includes the following components:
+- Apache Mesos 1.5.x [change log](https://github.com/apache/mesos/blob/2ead30d/CHANGELOG).
+- Marathon 1.6.564 [change log](https://github.com/mesosphere/marathon/tree/3fa693b32).
+- Metronome 0.4.5 [change log](https://github.com/dcos/metronome/releases/tag/v0.4.5).
+
+# Issues fixed in DC/OS 1.11.8
+
+## Docker containers
+- COPS-4087	- For applications that use Docker containers with a Virtual IP address, backend port mapping resolves access to the application by using the `host_IP:port_number` instead of the `container_ip:port_number`.
+
+## GUI
+- COPS-3369 - Updates to the DC/OS UI provide better rendering for elements such as environment variables, secrets, labels, and version information.   
+- DCOS_OSS-1961	- Pod lifecycle stages are supported in the DC/OS UI.
```

Comment:
Didn't we already have this issue documented in 1.11.7?  I checked the JIRA and it said it was merged twice, once into 1.11.8 and once into 1.11.7 -  but this can't be correct.

Reply:
@megatmar

Reply:
Do you mean COPS-3369 (which is linked to UI updates in its PR or JIRA thread) or DCOS_OSS-1961 (which was addressed in the 1.11.7 release notes but that description didn't say or imply anything to do with pod lifecycle)?

Reply:
@lsgunnlsgunn - is this COPS-3369 or should it be COPS-3360 (that's what i have in my notes)?
@yishan-lin - good catch.  i'll check with @orsenthil

Reply:
COPS-3360 - UI elements not rendering properly in DC/OS UI 
    COPS-3369 - Incorrect link in version policy doc (as far as I can tell the link is to the correct place)

Reply:
Changed to COPS-3360